src/sections/pools/stablepool/details/PoolDetails.tsx

```diff
@@ -0,0 +1,78 @@
+import { getAssetLogo } from "components/AssetIcon/AssetIcon"
+import { Separator } from "components/Separator/Separator"
+import { Text } from "components/Typography/Text/Text"
+import { useTranslation } from "react-i18next"
+import { MultipleIcons } from "components/MultipleIcons/MultipleIcons"
+import { Fragment } from "react"
+import { SBadge } from "../StablePool.styled"
+import { u32, u8 } from "@polkadot/types"
+import BigNumber from "bignumber.js"
+
+type PoolDetailsProps = {
+  assets: {
+    id: string
+    symbol: string
+    decimals: u8 | u32
+  }[]
+  tradeFee: BigNumber
+  className?: string
+}
+export const PoolDetails = ({
+  assets,
+  tradeFee,
+  className,
+}: PoolDetailsProps) => {
+  const { t } = useTranslation()
+
+  const tradeFeeDisplay = tradeFee.toString()
+
+  return (
+    <div sx={{ flex: "column" }} className={className}>
+      <div sx={{ flex: "row", justify: "space-between", align: "center" }}>
+        <div sx={{ flex: "column", gap: 26 }}>
+          <div sx={{ flex: "row", gap: 8, align: "center" }}>
+            <SBadge>
+              <Text fs={11} fw={700} color="basic900">
+                {t("liquidity.stablepool")}
+              </Text>
+            </SBadge>
+            <Text fs={13} color="basic400">
+              {t("liquidity.assets.title")}
+            </Text>
+          </div>
+          <div sx={{ flex: "column", gap: 5 }}>
+            <MultipleIcons
+              icons={assets.map((asset) => ({
+                icon: getAssetLogo(asset.symbol),
+              }))}
+            />
+            <div sx={{ flex: "row" }}>
+              {assets.map((asset, index) => (
+                <Fragment key={asset.symbol}>
+                  {index ? <Text color="whiteish500">/</Text> : null}
```

Comment:
what is it supposed to render?

Reply:
It is rendering `/` (slash) between assets 

![Screenshot 2023-08-22 at 7 58 09](https://github.com/galacticcouncil/HydraDX-ui/assets/19366687/8bc2bd0d-9452-4f3e-9e7e-65e37ad70be8)